File: help/pt_pt/exercicios_da_ficha_11/Ficha11_010/Ficha11_010.py
# Programador           -


#::::::::::::::::::::::::::::::::::::::::::::::::::
#User Defined Functions Code
#::::::::::::::::::::::::::::::::::::::::::::::::::
def linha(n):
    for i in range(0, n, 1 ):
        print( "*" , end="")
    print()

#::::::::::::::::::::::::::::::::::::::::::::::::::
def quadrado(altura):
    for i in range(1, altura + 1, 1 ):
        linha(altura )

if __name__ == "__main__":
    altura =  int(input("Altura : "))
    quadrado(altura )


#Python 3  - Automatic translated by Algorithmi 22.05
#(c) Ant0nio M@nso 	quarta, 2022 out 12 20:11

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_010/Ficha11_010.js
// Programador           -

//Main Function
function main() {
    document.write("Altura : " );// print text
    altura = parseInt(prompt("Altura : " ,"0") );
    
    document.write(altura + "\n");// print input 
    quadrado(altura ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function linha(n) {
    for( var i = 0; i < n; i = i + 1 ) {
        document.write("*" );
    }
    document.write("\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function quadrado(altura) {
    for( var i = 1; i < altura + 1; i = i + 1 ) {
        linha(altura ) ;
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:11
